python/fledge/services/core/server.py

```diff
@@ -504,15 +524,41 @@ async def installation_config(cls):
             category = 'Installation'
 
             if cls._configuration_manager is None:
-                _logger.error("No configuration manager available")
+                cls._logger.error("No configuration manager available")
             await cls._configuration_manager.create_category(category, config, 'Installation', True,
                                                              display_name='Installation')
             await cls._configuration_manager.get_category_all_items(category)
 
             cls._package_cache_manager = {"update": {"last_accessed_time": ""},
                                           "upgrade": {"last_accessed_time": ""}, "list": {"last_accessed_time": ""}}
         except Exception as ex:
-            _logger.exception(str(ex))
+            cls._logger.exception(str(ex))
+            raise
+
+    @classmethod
+    async def log_config(cls):
+        """ Get the logging level configuration """
+        try:
+            config = cls._LOGGING_DEFAULT_CONFIG
+            category = 'LOGGING'
+            description = "Logging Level of Core Server"
+            if cls._configuration_manager is None:
+                cls._logger.error("No configuration manager available.")
+            await cls._configuration_manager.create_category(category, config, description, True,
+                                                             display_name='Logging')
+            config = await cls._configuration_manager.get_category_all_items(category)
+            log_level = config['logLevel']['value']
+            if log_level == 'debug':
+                logging_level = logging.DEBUG
+            elif log_level == 'info':
+                logging_level = logging.INFO
+            elif log_level == 'warning':
+                logging_level = logging.WARNING
+            else:
+                logging_level = logging.ERROR
+            cls._log_level = logging_level
+        except Exception as ex:
+            cls._logger.exception(str(ex))
```

Comment:
in server py module, play with old logger module instead of the new one. This exception could be self destroying for that Logger class instance too.